tm/src/main/java/io/seata/tm/api/Propagation.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.tm.api;
+
+/**
+ * Propagation level of global transactions.
+ *
+ * @author haozhibei
+ */
+public enum Propagation {
```

Comment:
Propagation is a property of the transaction, can be at the same directory as TransactionInfo.java.

Reply:
ACK